uncontrast_studies/models/stimulus.py

```diff
@@ -86,7 +88,13 @@ class Meta:
         on_delete=CASCADE,
         related_name="target_stimuli",
     )
-
+    suppressed_stimulus = models.ForeignKey(null=False, blank=False, on_delete=PROTECT, to=UnConSuppressedStimulus)
+    is_target_same_as_suppressed_stimulus = models.BooleanField(
```

Comment:
shouldn't this be on the "suppressed" side?

Reply:
I don't think so, because it holds information about the target (in relation to the suppressed stimulus, via Foreign Key field). But I can change it if you think it's wrong